src/lib/airtable/request.js

```diff
@@ -646,6 +665,31 @@ export const updateManyInventorys = async (recordUpdates) => {
   return Promise.all(updatePromises);
 };
 
+// NONGENERATED: Review purchase request
+export const reviewPurchaseRequest = async (purchaseRequest) => {
+  try {
+    // Extract details relevant to reviewing the purchase request
+    const reviewData = {
+      reviewerId: purchaseRequest.reviewerId,
+      approvedAt: purchaseRequest.approvedAt,
+      status: purchaseRequest.status,
+    }
+    updatePurchaseRequest(purchaseRequest.id, reviewData);
```

Comment:
I originally had this as a custom endpoint (see backend PR) but switched it to this -- do you think this is fine for this use case?

Reply:
We might also want to set the precedent that "Purchase requests must have been persisted to Airtable before they can be approved" (The only workflow that this would cut off is an admin submitted a purchase request and self-approving it).

We might want to do this because the workflow might get too complicated for the workflow of:
1. Offline inventory created
2. While still offline, a purchase request is created
3. While still offline, that same purchase request (which doesn't have a valid id) gets reviewed